frame/composable-traits/src/clearing_house.rs

```diff
@@ -37,38 +37,38 @@ pub trait ClearingHouse {
 		amount: Self::Balance,
 	) -> Result<(), DispatchError>;
 
-	/// Create a new perpetuals market
+	/// Create a new perpetuals market.
 	///
 	/// ## Parameters
 	/// - `config`: specification for market creation
 	///
 	/// ## Returns
-	/// The new market's id, if successful
+	/// The new market's id, if successful.
 	fn create_market(config: &Self::MarketConfig) -> Result<Self::MarketId, DispatchError>;
 
-	/// Open a position in a market
+	/// Open a position in a market.
 	///
 	/// This may result in the following outcomes:
```

Comment:
I would change to "This will result in one of the following outcomes:"

Reply:
Fair enough, exactly one of the outcomes will happen if the call is successful